server/index.ts

```diff
@@ -152,7 +152,7 @@ onClientCallback('ox_banking:getDashboardData', async (playerId): Promise<Dashbo
     }[]
   >(
     `
-    SELECT amount, date, toId, fromId, message
+    SELECT amount, DATE_FORMAT(date, '%Y-%m-%d %H:00') as date, toId, fromId, message
```

Comment:
> %H:00

We don't particularly care about the exact time the transaction was sent at so I think we can get rid of that.